Update Edge operator overview with clearer positioning and IT security benefit (#4902)
SoftwareAG/c8y-docs: content/edge/edge-introduction-bundle/edge-operator.md

layout: redirect
---

> Edge is a single-server variant of the {{< product-c8y-iot >}} platform designed for easy, self-service operations, offering the flexibility to deploy anywhere from a standard local network to fully offline, air-gapped environments. Because it shares the same architecture as the {{< product-c8y-iot >}} cloud, it enables a "develop once, deploy anywhere" approach, allowing cloud-built solutions, customizations, and extensions to be seamlessly ported to the edge.

Reviewer: ```suggestion
Edge is a single-server deployment of the {{< product-c8y-iot >}} platform, designed for simple deployment and self-service operation. It can be deployed anywhere, from a standard local network to fully offline, air-gapped environments. Because it shares the same architecture as the {{< product-c8y-iot >}} cloud, it enables a "develop once, deploy anywhere" approach, allowing cloud-developed solutions, customizations, and extensions to be seamlessly deployed at the edge.
```
Changed the statement "to be seamlessly ported to the edge" to "to be seamlessly deployed at the edge". "Ported" can imply that code changes are required.

> Edge is a single-server variant of the {{< product-c8y-iot >}} platform designed for easy, self-service operations, offering the flexibility to deploy anywhere from a standard local network to fully offline, air-gapped environments. Because it shares the exact same architecture as the {{< product-c8y-iot >}} cloud, it enables a "develop once, deploy anywhere" approach where cloud-built solutions, customizations, and extensions can be seamlessly ported to the edge.

Reviewer 2: ```suggestion
Edge is a single-server variant of the {{< product-c8y-iot >}} platform designed for easy, self-service operations, offering the flexibility to deploy anywhere from a standard local network to fully offline, air-gapped environments. Because it shares the same architecture as the {{< product-c8y-iot >}} cloud, it enables a "develop once, deploy anywhere" approach, allowing cloud-built solutions, customizations, and extensions to be seamlessly ported to the edge.
```

Reasons for using an onsite installation of Edge include:

* **Autonomy**: Even if there is no cloud connection, tasks like data collection and data analysis can still be performed.
* **Data reduction**: Data is analyzed and aggregated close to assets, and thus less data needs to be sent to the cloud.
* **Reactivity**: Both Edge and the {{< product-c8y-iot >}} platform include real-time streaming analytics engines. However, placing the rule execution in Edge reduces latency, because the round-trip to cloud is omitted.

> * **IT Security**: Processing and storing data locally keeps sensitive information strictly within your own network perimeter, reducing exposure to external threats and simplifying compliance with strict data privacy regulations.

Reviewer: ```suggestion
* **IT Security**: Processing and storing data locally keeps sensitive information strictly within your own network perimeter, reducing exposure to external threats and helping meet data sovereignty and regulatory requirements.
```
